notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -807,7 +765,9 @@
         "\n",
         "By default, pipeline step caching is enabled. This means that the results of previous step executions are reused when possible.\n",
         "\n",
-        "if you want to disable caching for a pipeline run, you can pass the `enable_caching=False` argument to the `PipelineJob` constructor when you submit the pipeline job, as shown below.  Try submitting the example pipeline job again, first with and then without caching enabled."
+        "if you want to disable caching for a pipeline run, you can set the `enable_caching` parameter as **False** when creating the `PipelineJob` object. \n",
+        "\n",
+        "Try submitting the pipeline job twice: once with `enable_caching` set to True, and the other time with it set to False."
```

Comment:
suggested edit (to avoid using "it"):

- Try submitting the pipeline job twice: once with `enable_caching` set to True, and the other time with `enable_caching` set to False